identity_base_config/files/default/id-rails-console

```diff
@@ -40,6 +44,24 @@ if [ $# -ge 1 ]; then
     esac
 fi
 
+while true; do
+    read -r -p "Please provide an explanation for this Rails console: " USE_CASE
+    if [ ! -z "$USE_CASE" ]; then
+        break
+    fi
+done
+
+CHANNEL=$(cat /etc/login.gov/keys/slackchannel) || ((missing_keys++))
+WEBHOOK=$(cat /etc/login.gov/keys/slackwebhook) || ((missing_keys++))
+if [ ${missing_keys} -gt 0 ]; then
+    echo 'Slack channel/webhook missing! Verify files and run Chef to repopulate.'
+    exit 1
+fi
+
+TEXT=":rails:*rails console*:rails: for \`$(logname)\` on \`$(hostname)\`\nuse case: \\\"${USE_CASE}\\\""
+PAYLOAD="'payload={\"channel\": \"${CHANNEL}\", \"username\": \"id-rails-console\", \"text\": \"${TEXT}\", \"icon_emoji\": \":dopetopus:\"}'"
+curl -X POST --data-urlencode "${PAYLOAD} ${WEBHOOK}"
```

Comment:
Shouldn't payload and webhook should be separate arguments?

Reply:
I was thinking of the previous behavior that utilized a full string for the command; will correct.